src/Core/src/Exception/UnexpectedValue.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Core\Exception;
+
+class UnexpectedValue extends \UnexpectedValueException implements Exception
```

Comment:
`UnexpectedValueException`

Reply:
Are you sure? We usually dont use "Exception" suffix. 

<img width="253" alt="Screenshot 2020-11-03 at 11 23 46" src="https://user-images.githubusercontent.com/1275206/97973895-14d2f880-1dc7-11eb-8043-6aaadacad34f.png">

Reply:
ohh.. I was trapped by `RuntimeException` and `LogicException`... We are not consistent :cry:

Reply:
True. But what we currently have sounds best. 

This is not super nice: 
```php
throw new Logic('Wrong');
```